demos/demo_time_series.cpp

```diff
@@ -0,0 +1,54 @@
+/**
+ * @file demo_time_series.cpp
+ * @author Vincent Berenz
+ * @copyright Copyright (c) 2019, Max Planck Gesellshaft.
+ *
+ * @brief basic usage of time series
+ *
+ */
+
+#include "real_time_tools/thread.hpp"
+#include "time_series/time_series.hpp"
+
+bool RUNNING = true;
```

Comment:
all-caps should only be used for constants.  Global variables should be prefixed with `g_` according to our [style guide](https://machines-in-motion.github.io/code_documentation/ci_example_cpp/coding_guidelines_1.html).

Reply:
+1